### PR TITLE
Add telemetry (pushing event logs back to purecloud)

### PR DESCRIPTION
**Pull request by xdumaine**

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -150,6 +178,61 @@ class PureCloudWebrtcSdk extends WildEmitter {
...
+        appId: 'webrtc-sdk',
+        appVersion: '[AIV]{version}[/AIV]' // injected by webpack auto-inject-version
+      },
+      traces
+    };
+    return requestApi.call(this, '/diagnostics/trace', {
+      method: 'post',
+      contentType: 'application/json; charset=UTF-8',
+      data: JSON.stringify(payload)
+    }).catch(() => {
```
Comment (jensengar) [inline src/client.js]: are we not going to retry or implement exponential backoff?

File: src/client.js [hunk 2]
```diff
@@ -150,6 +178,61 @@ class PureCloudWebrtcSdk extends WildEmitter {
...
+  _log (level, message, details) {
+    // immediately log it locally
+    this.logger[level](message, details);
+
+    // ex: if level is debug and config is warn, then debug is less than warn, don't push
+    if (LOG_LEVELS.indexOf(level) < LOG_LEVELS.indexOf(this._logLevel)) {
+      return;
+    }
+
+    if (this._optOutOfTelemetry) {
```
Comment (omarestrella) [inline src/client.js]: How do you actually opt out here? Not seeing a way to supply via options (meant to be kept private for internal use?)

Comment (xdumaine) [reply src/client.js]: Yeah, you can just set this directly on the sdk instance. Meant to be a little hidden.

File: src/client.js (hunk 1 above)
Comment (xdumaine) [reply src/client.js]: I thought about that. I think I'd like to do that in another version if necessary. Logs from this shouldn't be critical and should be pretty low in volume.

**Pull request merged**